source/Halibut/Transport/Protocol/TemporaryFileStream.cs

```diff
@@ -33,12 +42,14 @@ void SetFilePermissionsToInheritFromParent(string filePath)
             try
             {
                 var fileInfo = new FileInfo(filePath);
+#pragma warning disable PC001 // API not supported on all platforms
                 var fileSecurity = fileInfo.GetAccessControl();
 
                 //When isProtected (first param) is false, SetAccessRuleProtection changes the permissions of the file to allow inherited permissions. 
                 //preserveInheritance (second param) is ignored when isProtected is false.
                 fileSecurity.SetAccessRuleProtection(false, false);
                 fileInfo.SetAccessControl(fileSecurity);
+#pragma warning restore PC001 // API not supported on all platforms
             }
             catch (PlatformNotSupportedException)
```

Comment:
You can probably take this out now. This was a work around for the same problem

Reply:
fixed